.idea/checkstyle-idea.xml

```diff
@@ -3,13 +3,13 @@
   <component name="CheckStyle-IDEA">
     <option name="configuration">
       <map>
-        <entry key="active-configuration" value="PROJECT_RELATIVE:$PRJ_DIR$/gradle/codequality/checkstyle.xml:Record Layer Checks" />
+        <entry key="active-configuration" value="PROJECT_RELATIVE:$PROJECT_DIR$/gradle/codequality/checkstyle.xml:Record Layer Checks" />
         <entry key="checkstyle-version" value="7.5.1" />
         <entry key="copy-libs" value="false" />
         <entry key="location-0" value="BUNDLED:(bundled):Sun Checks" />
         <entry key="location-1" value="BUNDLED:(bundled):Google Checks" />
-        <entry key="location-2" value="PROJECT_RELATIVE:$PRJ_DIR$/gradle/codequality/checkstyle.xml:Record Layer Checks" />
-        <entry key="location-3" value="PROJECT_RELATIVE:$PRJ_DIR$/gradle/codequality/checkstyle-mandatory.xml:Record Layer Mandatory" />
+        <entry key="location-2" value="PROJECT_RELATIVE:$PROJECT_DIR$/gradle/codequality/checkstyle.xml:Record Layer Checks" />
+        <entry key="location-3" value="PROJECT_RELATIVE:$PROJECT_DIR$/gradle/codequality/checkstyle-mandatory.xml:Record Layer Mandatory" />
```

Comment:
Hm, I don't think we want this change (at least not here). I don't *think* this change is problematic, but I don't actually know.